### PR TITLE
fix: set HttpPackageHandler cache dir to temp dir

### PR DESCRIPTION
**Pull request by samsze1**
+ Logger basic config and logging message

ED-1035

### CODE REVIEW

**Comment by samsze1** [comment]
❤️ @jackylauu

File: src/dismantle/package/_handlers.py [hunk 1]
```diff
@@ -282,7 +281,10 @@ def _fetch_and_extract(self):
...
             raise FileNotFoundError(req.status_code)
         elif req.status_code == 200:
-            self._cache.parents[0].mkdir(parents=True, exist_ok=True)
+            # Make sure parent folder of the cache exists
+            self._cache.parents[0].mkdir(0o777, parents=True, exist_ok=True)
+            log.info(f'Creating dir {self._cache.parents[0]} to hold cache')
+            # Write the package raw bytes into a single cache file
             with open(self._cache, 'wb') as cached_package:
                 cached_package.write(req.content)
             self._updated = True
```
Comment (vayan) [inline src/dismantle/package/_handlers.py:290]: Thanks! can you also bump the version to `1.1.0` in `src/dismantle/__version__.py`?

**Review by vayan — APPROVED** [review]

**Comment by samsze1** [comment]
@vayan Should I just leave the windows unit test "unpassed"?

**Comment by vayan** [comment]
yeah don't bother 😄

**Pull request merged**